### DISCUSSION TITLE
主题开发时，安装plugin-docsearch后，eslint检查*.vue出现 react语法错误提示

### DISCUSSION THREAD

**Discussion by pengzhanbo** [vuepress/core#779] [Q&A] [answered]
## 描述
在进行theme开发时，我参考`vuepress-next`项目进行了eslint配置。

但是当我安装了`@vuepress/plugin-docsearch`插件后，在`*.vue`文件中，eslint开始提示我 react语法相关的错误提示。

当我卸载了`@vuepress/plugin-docsearch`插件，则不再出现提示。
 
我自己阅读了

- `eslint-config-vuepress`
- `eslint-config-vuepress-typescript`
-  `prettier-config-vuepress` 
- `vuepress-next `的 `.eslintrc.js` 文件

但没有找到问题的原因在哪里。

**Comment by Mister-Hope** [comment, marked as answer]
tsconfig里把react的type排除掉
